cmd/snaplock/runinhibit/inhibit.go

```diff
@@ -132,7 +138,13 @@ func removeInhibitInfoFiles(snapName string) error {
 // start and will block, presenting a user interface if possible. Also
 // info.Previous corresponding to the snap revision that was installed must be
 // provided and cannot be unset.
-func LockWithHint(snapName string, hint Hint, info InhibitInfo) error {
+func LockWithHint(snapName string, hint Hint, info InhibitInfo, unlocker Unlocker) error {
```

Comment:
doc comments need to be updated to mention unlocker

Reply:
updated